content/en/opentelemetry/guide/switch_from_processor_to_connector.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Switch from Datadog Processor to Datadog Connector for OpenTelemetry APM Metrics
+kind: guide
+is_beta: true
+further_reading:
+- link: "/opentelemetry/guide/ingestion_sampling_with_opentelemetry/"
+  tag: "Documentation"
+  text: "Ingestion Sampling with OpenTelemetry"
+- link: "https://opentelemetry.io/docs/collector/build-connector/"
+  tag: "OpenTelemetry"
+  text: "Connectors in OpenTelemetry"
+---
+
+{{< callout url="" header="false" btn_hidden="true" >}}
+  Datadog connector for the OpenTelemetry Collector is in beta. 
+{{< /callout >}} 
+
+## Overview
+
+If you have a previously setup OpenTelemetry Collector pipeline that prepends the soon-to-be deprecated [Datadog processor][1] so that APM metrics are calculated from unsampled data, switch that pipeline to use the [Datadog connector][2] instead. 
+
+The Datadog connector for the OpenTelemetry Collector allows APM metrics to be calculated on 100% of the trace data without any sampling, sets the appropriate retention for the metrics it passes through, and complies with the OpenTelemetry standard better than the Datadog Processor does.
```

Comment:
I also don't think "sets the appropriate retention for the metrics" is a very clear sentence and could possibly be removed